devtools/git-subtree-check.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/sh
+# Copyright (c) 2015-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C
+
+check_remote=0
+while getopts "?hr" opt; do
+  case $opt in
+    '?' | h)
+      echo "Usage: $0 [-r] DIR [COMMIT]"
+      echo "       $0 -?"
+      echo ""
+      echo "Checks that a certain prefix is pure subtree, and optionally whether the"
+      echo "referenced commit is present in any fetched remote."
+      echo ""
+      echo "DIR is the prefix within the repository to check."
+      echo "COMMIT is the commit to check, if it is not provided, HEAD will be used."
+      echo ""
+      echo "-r      Check that subtree commit is present in repository."
+      echo "        To do this check, fetch the subtreed remote first. Example:"
+      echo ""
+      echo "            git fetch https://github.com/bitcoin-core/secp256k1.git"
+      echo "            test/lint/git-subtree-check.sh -r src/secp256k1"
```

Comment:
similar to https://github.com/PIVX-Project/PIVX/pull/2473#discussion_r670526683, we can maybe do
```suggestion
      echo "            git fetch https://github.com/relic-toolkit/relic.git"
      echo "            devtools/git-subtree-check.sh -r contrib/relic"
```

which btw checks out here
```
contrib/relic in HEAD currently refers to tree 23de0f0a5c484c4eaa440e14836819a791ea3e13
contrib/relic in HEAD was last updated in commit 914ee5252e232a7308c9d38d9c259f344a38ee51 (tree 23de0f0a5c484c4eaa440e14836819a791ea3e13)
contrib/relic in HEAD was last updated to upstream commit 1885ae3b681c423c72b65ce1fe70910142cf941c (tree 23de0f0a5c484c4eaa440e14836819a791ea3e13)
GOOD
```

Reply:
yeah, will update it. Same as the other one.